diff --git a/conf/machine/phyboard-pollux-imx8mp-3.conf b/conf/machine/phyboard-pollux-imx8mp-3.conf
index 2d26e7ad..33dedba3 100644
--- a/conf/machine/phyboard-pollux-imx8mp-3.conf
+++ b/conf/machine/phyboard-pollux-imx8mp-3.conf
@@ -13,6 +13,8 @@ MACHINE_FEATURES:append:use-nxp-bsp = " tpm2"
 
 KERNEL_DEVICETREE = " \
         freescale/imx8mp-phyboard-pollux-rdk.dtb \
+        freescale/imx8mp-phyboard-pollux-rdk-falcon-sdcard.dtb \
+        freescale/imx8mp-phyboard-pollux-rdk-falcon-emmc.dtb \
         freescale/overlays/imx8mp-isp-csi1.dtbo \
         freescale/overlays/imx8mp-isp-csi2.dtbo \
         freescale/overlays/imx8mp-isi-csi1.dtbo \
